src/main/resources/messages/RsBundle.properties

```diff
@@ -93,6 +93,18 @@ settings.rust.auto.import.title=Rust
 settings.rust.completion.suggest.out.of.scope.items=Suggest out-of-scope items
 settings.rust.completion.title=Rust
 
+settings.rust.inlay.hints.for.closures=Closures
+settings.rust.inlay.hints.for.iterators=Iterators as `impl Iterator<...>`
+settings.rust.inlay.hints.for.loop.variables=Loop variables
+settings.rust.inlay.hints.for.obvious.types=Obvious types
+settings.rust.inlay.hints.for.same.consecutive.types=Same consecutive types
+settings.rust.inlay.hints.for.type.placeholders=Type placeholders
+settings.rust.inlay.hints.for.variables=Variables
+settings.rust.inlay.hints.for=Show hints for:
+settings.rust.inlay.hints.title.method.chains=Method chains
+settings.rust.inlay.hints.title.types=Types
+settings.rust.inlay.parameter.hints.only.smart=Only smart hints
```

Comment:
@MarinaKalashina Sorry, it's not a large extraction but this PR is not only about user-visible strings extraction